webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const _ = require('lodash');
+const path = require('path');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const { tokens } = require('./lib/dsl/lexer');
+
+const allTokenTypeNames = _.mapValues(tokens, tokenType => tokenType.name);
+
+module.exports = {
+  mode: 'production',
+  entry: './lib/dsl/exports.js',
+  output: {
+    path: path.resolve(__dirname, './dist/'),
+    filename: 'jhipster.min.js',
```

Comment:
can we call this file `jdl-core.min.js` please

Reply:
yes will fix

Reply:
changed the file name to jdl-core.min.js **and** the library name
to match as "jdlCore".